test/core/test_lwt_mvar.ml

```diff
@@ -0,0 +1,73 @@
+(* OCaml promise library
+ * http://www.ocsigen.org/lwt
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, with linking exceptions;
+ * either version 2.1 of the License, or (at your option) any later
+ * version. See COPYING file for details.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
+ *)
+
+open Test
+open Lwt
```

Comment:
This is optional, but I recommend removing this (sorry if you saw it used in another test suite). Your code already nicely uses `Lwt.` in most places, removing it would, I think, just make the `return`s at the end of each test more explicit.